### ISSUE TITLE
Add KaOS to package list 

### ISSUE THREAD

**Issue by ShalokShalom** [curl/curl#1937]
http://kaosx.tk/packages/index.php?act=show&subdir=core&sortby=date&file=curl-7.55.1-1-x86_64.pkg.tar.xz

This here is a mirror list: 

```
# Default mirror, USA
Server = https://mirror.math.princeton.edu/pub/kaoslinux/$repo

# Germany
Server = http://mirror6.layerjet.com/kaos/$repo
Server = https://mirror.alpix.eu/kaos/$repo
Server = https://kaos-frankfurt.tk/repo/$repo

# Czech Republic
Server = http://193.86.184.157/kaos/$repo

# France
Server = http://kaos.youm.org/$repo
Server = http://mirror.lignux.com/kaos/$repo

# Netherlands
Server = http://nyxi.eu/mirrors/kaos/$repo

# South-Africa
Server = http://ftp.wa.co.za/pub/kaos/$repo

# United States
Server = http://mirror.dacentec.com/kaosx/$repo
Server = http://kaosx.tk/repo/$repo
Server = https://repo.kaosx.us/$repo
Server = https://kqtos.tk/repo/$repo

# Vietnam
Server = http://mirror.freedif.org/KaOS/$repo



```

**Comment by bagder** [comment]
Thanks.

Do you know if there's a convenient page that we can pull and scan for the latest curl version hosted there? The link about takes us to the 7.55.1 page but I search for curl and see that there's a 7.56.0 now.

**Comment by ShalokShalom** [comment]
Hi there and thanks a lot for your response. 7.55.1 is currently in use, 7.56.0 is in build repo:

http://kaosx.tk/packages/index.php?act=search&subdir=&sortby=date&order=descending&searchpattern=curl

Build repo is where all new packages land in the first place, straight after the first build of this version.
There it than stands for the next couple of days, usually a week or two. 

Then it comes in the app/core/main repo, where ever it belongs. Curl in the core repo, as you can see.

**Comment by bagder** [comment]
That URL is really hard to parse for a script to figure out which curl version it is that you're providing.

Without a page in which we can figure out the curl version, we'd have to rely on humans to update the info for every release and that is very tedious work that is destined to be forgotten or ignored at some point...

**Comment by ShalokShalom** [comment]
https://github.com/KaOSx/core/blob/master/curl/PKGBUILD
